samples/nrf_desktop/CMakeLists.txt

```diff
@@ -71,3 +74,27 @@ add_subdirectory(src/events)
 add_subdirectory(src/hw_interface)
 add_subdirectory(src/modules)
 add_subdirectory(src/services)
+
```

Comment:
below will add a command to sign hex file if you use
`ninja sign`

@SebastianBoe  is there a plan to add something like that in zephyr?

Reply:
Yes.

This is on it's way; https://github.com/zephyrproject-rtos/zephyr/commit/6679ff7647c29a553f835ac16b9680c0fd341be3#diff-93ebc7c448cc5ffd4336cc014cc05574R18

And also, another approach; https://github.com/zephyrproject-rtos/zephyr/pull/12903